Userland/Services/WebContent/BackingStoreManager.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/Page/Page.h>
+#include <WebContent/Forward.h>
+
+namespace WebContent {
+
+class BackingStoreManager {
+public:
+    void resize_backing_stores_if_needed(bool window_resize_in_progress);
```

Comment:
Is this a pure code movement commit? If not, a `Yes. No` or `Resizing, <other condition>` enum might make this easier to understand.